MobSF/utils.py

```diff
@@ -80,7 +88,7 @@ def check_update():
 Please update from master branch or check for new releases.\n""")
                 else:
                     print("\n[INFO] No updates available.")
-    except (urllib.error.HTTPError, http.client.HTTPException):
+    except (requests.exceptions.HTTPError, http.client.HTTPException):

```

Comment:
I think requests don't have a `http.client.HTTPException`. Not sure though.
So replace this with `except requests.exceptions.HTTPError as err:` probably.